src/main/scala/abalone/Role.scala

```diff
@@ -19,9 +19,11 @@ sealed trait Role {
 
 sealed trait PromotableRole extends Role
 
-case object Stone extends Role { // @TODO VFR: just use Stone for Marbles
+case object Stone extends Role { // @TODO VFR: consider if we want a new "Marble" Role or if we just use Stone for Marbles (because these can be rolling stones, not Oware "Stones")
```

Comment:
remove comment

Reply:
I do not see the comment being a problem until all this remains on 503-abalone, but I think you then confirm we do not want another Role, (to express the fact these pieces can move unlike in Oware or Go) ?

Reply:
I was considering renaming Stone to Marble, as the Stone we use in strategygames is usually static from what I know.
What do you think of that idea ?

Reply:
sure I dont think it matters.